### ISSUE TITLE
Chapter 6 - Broken image references in Subsections "FOR NON-DEVELOPERS", "FOR COMPLEX PROJECTS"

### ISSUE THREAD

**Issue by srivasud** [mschermann/data_viz_reader#1420]
The following line numbers have broken image reference (Either the image is corrupted or image name in the citation is wrong or missing image itself).

1) https://github.com/mschermann/data_viz_reader/blob/0dfc9468b402eeb69bf0c35880bf781007afdbac/06-process.Rmd#L552
2)
https://github.com/mschermann/data_viz_reader/blob/0dfc9468b402eeb69bf0c35880bf781007afdbac/06-process.Rmd#L555
3)
https://github.com/mschermann/data_viz_reader/blob/0dfc9468b402eeb69bf0c35880bf781007afdbac/06-process.Rmd#L559
4)
https://github.com/mschermann/data_viz_reader/blob/0dfc9468b402eeb69bf0c35880bf781007afdbac/06-process.Rmd#L562
5)
https://github.com/mschermann/data_viz_reader/blob/0dfc9468b402eeb69bf0c35880bf781007afdbac/06-process.Rmd#L566
6)
https://github.com/mschermann/data_viz_reader/blob/0dfc9468b402eeb69bf0c35880bf781007afdbac/06-process.Rmd#L570
7)
https://github.com/mschermann/data_viz_reader/blob/0dfc9468b402eeb69bf0c35880bf781007afdbac/06-process.Rmd#L578
8)
https://github.com/mschermann/data_viz_reader/blob/0dfc9468b402eeb69bf0c35880bf781007afdbac/06-process.Rmd#L581
9)
https://github.com/mschermann/data_viz_reader/blob/0dfc9468b402eeb69bf0c35880bf781007afdbac/06-process.Rmd#L585
10)
https://github.com/mschermann/data_viz_reader/blob/0dfc9468b402eeb69bf0c35880bf781007afdbac/06-process.Rmd#L588

**Comment by XZou0803** [comment]
Good catch. I think it is a formatting problem and it can be fixed by just deleting the contexts inside the blanket"[]"

**Comment by srivasud** [comment]
Potential Solution
==============
1) The image references are wrong. The filenames are missing the file extension.
2) I noticed one or two image files are corrupted and few files are missing.
Missing files
==========
1) ggplot2s.png
Corrupted files
=========
1) NetworkX.png
2) Seaborn.png
3)ggvis.png
4)Bokeh.png
